R/deparse.R

```diff
@@ -655,9 +655,9 @@ list_deparse <- function(x, lines = new_lines()) {
   lines$push(paste0("<list: "))
   lines$increase_indent()
 
-  truncated <- length(x) > 5L
+  truncated <- !is.null(max_len) && length(x) > max_len
   if (truncated) {
-    x <- .subset(x, 1:5)
+    x <- .subset(x, 1:max_len)
```

Comment:
Should be `seq_len()` in case `max_len` is 0.

Reply:
Done